proposals/solid-oidc-primer/index.bs

```diff
@@ -817,78 +798,55 @@ GET https://secureauth.example/.well-known/openid-configuration
 
 Response
 ```json
+```json
```

Comment:
```suggestion
```

You've doubled the code-section start
```
```json
```

Reply:
Good spot, thanks!